src/structures/SimpleOpenable.ts

```diff
@@ -15,8 +15,9 @@ export default class SimpleOpenable extends Openable {
 		this.table = options.table;
 	}
 
-	public open(quantity = 1): ItemBank {
+	public open(quantity = 1, options: OpenableOpenOptions = {}): ItemBank {
 		const loot = new Loot();
+		options;
```

Comment:
what is this doing

Reply:
It does nothing, it was there beforehand I think. Removing it.